pkg/kubelet/kuberuntime/instrumented_services_test.go

```diff
@@ -89,3 +90,112 @@ func TestStatus(t *testing.T) {
 	}
 	assert.Equal(t, expected, actural)
 }
+
+func TestInstrumentedListContainer(t *testing.T) {
+	fakeRuntime, _, _, _ := createTestRuntimeManager()
+	irs := newInstrumentedRuntimeService(fakeRuntime)
+	podName, namespace := "foo", "bar"
+	containerName, image := "sidecar", "logger"
+	configs := []*runtimeapi.ContainerConfig{}
+	sConfigs := []*runtimeapi.PodSandboxConfig{}
+	for i := 0; i < 3; i++ {
+		s := makeSandboxConfig(fmt.Sprintf("%s%d", podName, i),
```

Comment:
I think it would be more readable not to use helper methods and constants, since each is only used once.  Just write out the structs here.

Reply:
Done.